js/base-chat.js

```diff
@@ -569,6 +569,18 @@ function baseChat () {
 				"_type": "boolean",
 				"_player": true,
 			},
+			"executeJSMacro": {
+				"name": "Execute JS script in macros",
+				"default": "own",
+				"_type": "_enum",
+				"__values": ["none", "own", "all"],
+				"__texts": [
+					"disabled",
```

Comment:
nit: capitalise the first letter of these

Reply:
done